### PR TITLE
CLI: set maxErrors to Infinity for autoconfigure

### LINKED ISSUE #1695: auto-configure only analyzes presets against 50 error
Hello,
I tried to run auto-configure against a single file to have it analyzed against the preset profiles. In the results it says only 50 errors but there are at least over 5k. Adding -m 100000 didn't do anything at all. Also adding maxerrors in a config file didn't help. It seems to be hard coded somewhere.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25903528-auto-configure-only-analyzes-presets-against-50-error?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by hzoo**
Fixes #1695

### CODE REVIEW

**Comment by mrjoelkemp** [comment]
LGTM

File: lib/cli.js [hunk 1]
```diff
@@ -82,8 +82,8 @@ module.exports = function cli(program) {
     }
 
     // It is more logical to set maxErrors to Infinity since
-    // it doesn't makes sense to limit error output with enabled "fix" option
-    if (program.fix) {
+    // it doesn't makes sense to limit error output with enabled "fix"/"autoconfigure" option
```
Comment (mikesherov) [inline lib/cli.js]: Explain why or remove comment.

**Comment by hzoo** [comment]
I'd be ok with removing the comment but I added one

**Comment by markelog** [comment]
It would be cool to take down https://github.com/jscs-dev/node-jscs/issues/1722 while we at it

**Pull request closed without merging**